mitodl/micromasters
# Generate robotic avatars

#### What are the relevant tickets?
Fixes #915

#### What's this PR do?
Adds a management command `attach_avatars` with argument `--name-prefix` to attach robot avatars to profiles.

#### How should this be manually tested?
Look at the learner search. Choose a username prefix and run the management command. Adjust the search or refresh the page and note that the user avatar is now a robot

#### Any background context you want to provide?
This uses the robohash library to generate the avatars based on the username, it doesn't actually download it from https://robohash.com

#### Screenshots (if appropriate)
![screenshot from 2017-03-17 16-37-02](https://cloud.githubusercontent.com/assets/863262/24061978/0992b42c-0b30-11e7-8bdf-a83d9bd9a64e.png)
